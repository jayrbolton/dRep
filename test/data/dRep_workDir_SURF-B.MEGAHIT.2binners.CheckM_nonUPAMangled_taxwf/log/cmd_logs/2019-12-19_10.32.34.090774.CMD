prodigal -i /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.012.fasta -d /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.012.fasta.fna -a /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.012.fasta.faa -m -p meta
