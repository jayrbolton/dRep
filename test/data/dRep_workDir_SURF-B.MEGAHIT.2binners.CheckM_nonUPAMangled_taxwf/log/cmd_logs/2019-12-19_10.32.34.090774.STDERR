-------------------------------------
PRODIGAL v2.6.3 [February, 2016]         
Univ of Tenn / Oak Ridge National Lab
Doug Hyatt, Loren Hauser, et al.     
-------------------------------------
Request:  Metagenomic, Phase:  Training
Initializing training files...done!
-------------------------------------
Request:  Metagenomic, Phase:  Gene Finding
Finding genes in sequence #1 (28690 bp)...done!
Finding genes in sequence #2 (7378 bp)...done!
Finding genes in sequence #3 (43200 bp)...done!
Finding genes in sequence #4 (21016 bp)...done!
Finding genes in sequence #5 (35950 bp)...done!
Finding genes in sequence #6 (42019 bp)...done!
Finding genes in sequence #7 (10466 bp)...done!
Finding genes in sequence #8 (44305 bp)...done!
Finding genes in sequence #9 (34523 bp)...done!
Finding genes in sequence #10 (35068 bp)...done!
Finding genes in sequence #11 (3640 bp)...done!
Finding genes in sequence #12 (192394 bp)...done!
Finding genes in sequence #13 (83851 bp)...done!
Finding genes in sequence #14 (16201 bp)...done!
Finding genes in sequence #15 (34497 bp)...done!
Finding genes in sequence #16 (7818 bp)...done!
Finding genes in sequence #17 (26028 bp)...done!
Finding genes in sequence #18 (10451 bp)...done!
Finding genes in sequence #19 (7018 bp)...done!
Finding genes in sequence #20 (66186 bp)...done!
Finding genes in sequence #21 (38140 bp)...done!
Finding genes in sequence #22 (80129 bp)...done!
Finding genes in sequence #23 (5057 bp)...done!
Finding genes in sequence #24 (15952 bp)...done!
Finding genes in sequence #25 (14304 bp)...done!
Finding genes in sequence #26 (6324 bp)...done!
Finding genes in sequence #27 (49170 bp)...done!
Finding genes in sequence #28 (3761 bp)...done!
Finding genes in sequence #29 (11239 bp)...done!
Finding genes in sequence #30 (110762 bp)...done!
Finding genes in sequence #31 (65568 bp)...done!
Finding genes in sequence #32 (42178 bp)...done!
Finding genes in sequence #33 (32915 bp)...done!
Finding genes in sequence #34 (26126 bp)...done!
Finding genes in sequence #35 (16995 bp)...done!
Finding genes in sequence #36 (9327 bp)...done!
Finding genes in sequence #37 (19938 bp)...done!
Finding genes in sequence #38 (3828 bp)...done!
Finding genes in sequence #39 (73446 bp)...done!
Finding genes in sequence #40 (42229 bp)...done!
Finding genes in sequence #41 (27840 bp)...done!
Finding genes in sequence #42 (33092 bp)...done!
Finding genes in sequence #43 (32639 bp)...done!
Finding genes in sequence #44 (27034 bp)...done!
Finding genes in sequence #45 (31431 bp)...done!
Finding genes in sequence #46 (24726 bp)...done!
Finding genes in sequence #47 (18007 bp)...done!
Finding genes in sequence #48 (12057 bp)...done!
Finding genes in sequence #49 (15965 bp)...done!
Finding genes in sequence #50 (48186 bp)...done!
Finding genes in sequence #51 (7328 bp)...done!
Finding genes in sequence #52 (75579 bp)...done!
Finding genes in sequence #53 (8979 bp)...done!
Finding genes in sequence #54 (55704 bp)...done!
Finding genes in sequence #55 (54608 bp)...done!
Finding genes in sequence #56 (6995 bp)...done!
Finding genes in sequence #57 (39582 bp)...done!
Finding genes in sequence #58 (31599 bp)...done!
Finding genes in sequence #59 (6013 bp)...done!
Finding genes in sequence #60 (18827 bp)...done!
Finding genes in sequence #61 (19084 bp)...done!
Finding genes in sequence #62 (9260 bp)...done!
Finding genes in sequence #63 (64980 bp)...done!
Finding genes in sequence #64 (27622 bp)...done!
Finding genes in sequence #65 (7559 bp)...done!
Finding genes in sequence #66 (3987 bp)...done!
Finding genes in sequence #67 (23624 bp)...done!
Finding genes in sequence #68 (63074 bp)...done!
Finding genes in sequence #69 (22700 bp)...done!
Finding genes in sequence #70 (50885 bp)...done!
Finding genes in sequence #71 (18203 bp)...done!
Finding genes in sequence #72 (74784 bp)...done!
Finding genes in sequence #73 (27543 bp)...done!
Finding genes in sequence #74 (12795 bp)...done!
Finding genes in sequence #75 (9854 bp)...done!
Finding genes in sequence #76 (21006 bp)...done!
Finding genes in sequence #77 (9354 bp)...done!
Finding genes in sequence #78 (75929 bp)...done!
Finding genes in sequence #79 (45420 bp)...done!
Finding genes in sequence #80 (37368 bp)...done!
Finding genes in sequence #81 (85569 bp)...done!
Finding genes in sequence #82 (57618 bp)...done!
Finding genes in sequence #83 (20795 bp)...done!
Finding genes in sequence #84 (13140 bp)...done!
Finding genes in sequence #85 (61893 bp)...done!
Finding genes in sequence #86 (103804 bp)...done!
Finding genes in sequence #87 (18844 bp)...done!
Finding genes in sequence #88 (35071 bp)...done!
Finding genes in sequence #89 (52070 bp)...done!
Finding genes in sequence #90 (23735 bp)...done!
Finding genes in sequence #91 (11644 bp)...done!
Finding genes in sequence #92 (75072 bp)...done!
Finding genes in sequence #93 (5377 bp)...done!
Finding genes in sequence #94 (21883 bp)...done!
Finding genes in sequence #95 (33394 bp)...done!
Finding genes in sequence #96 (3958 bp)...done!
Finding genes in sequence #97 (14116 bp)...done!
Finding genes in sequence #98 (40635 bp)...done!
Finding genes in sequence #99 (9917 bp)...done!
Finding genes in sequence #100 (2737 bp)...done!
Finding genes in sequence #101 (7951 bp)...done!
Finding genes in sequence #102 (22280 bp)...done!
Finding genes in sequence #103 (95804 bp)...done!
Finding genes in sequence #104 (33240 bp)...done!
Finding genes in sequence #105 (5477 bp)...done!
Finding genes in sequence #106 (60881 bp)...done!
Finding genes in sequence #107 (15109 bp)...done!
Finding genes in sequence #108 (48722 bp)...done!
Finding genes in sequence #109 (61491 bp)...done!
Finding genes in sequence #110 (24609 bp)...done!
Finding genes in sequence #111 (26368 bp)...done!
Finding genes in sequence #112 (32145 bp)...done!
Finding genes in sequence #113 (6502 bp)...done!
Finding genes in sequence #114 (37323 bp)...done!
Finding genes in sequence #115 (8176 bp)...done!
Finding genes in sequence #116 (22075 bp)...done!
Finding genes in sequence #117 (23693 bp)...done!
Finding genes in sequence #118 (22906 bp)...done!
Finding genes in sequence #119 (23146 bp)...done!
Finding genes in sequence #120 (16893 bp)...done!
Finding genes in sequence #121 (102226 bp)...done!
Finding genes in sequence #122 (10110 bp)...done!
Finding genes in sequence #123 (33790 bp)...done!
